app/src/main/java/ru/prometeydev/movie/model/domain/ActorDto.kt

```diff
@@ -0,0 +1,11 @@
+package ru.prometeydev.movie.model.domain
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class ActorDto(
+    @Json(name = "id") val id: Int,
```

Comment:
This is my habit. I read somewhere about such a recommendation to annotate each field. I really don’t know how bestpracticles are.

Reply:
Can you provide a link? The moshi will generate the adapter and it will have correct string for parsing so you will not have any problem with obfuscation. As for me it is noise - I can not see which filed on BE are different from my model. 

But up to you. Your style is yours. But you have to have good grounded explanation why you do it.